app/forms/flood_risk_engine/steps/local_authority_address_form.rb

```diff
@@ -4,7 +4,9 @@ module Steps
     class LocalAuthorityAddressForm < BaseAddressForm
 
       def self.factory(enrollment)
-        raise(FormObjectError, "No Organisation set for step #{enrollment.current_step}") unless enrollment.organisation
+        unless enrollment.organisation
```

Comment:
normally agree, but my editor flagged this line as over 120 chars long so changed it